client/src/components/setup/FormFirstPage/FormFields.js

```diff
@@ -1,4 +1,4 @@
-import RenderField from '../RenderField/RenderField';
+import RenderField from '../../utils/RenderField/RenderField';
 
```

Comment:
Moved RenderField to another location because it's used NOT just by the /setup folder